### ISSUE TITLE
[ARRM Survey Results] [4. ARRM Framework Introduction]

### ISSUE THREAD

**Issue by JenniferChadwick** [w3c/wai-roles-responsbilities#41]
[Current text on ARRM Introduction page](https://www.w3.org/WAI/EO/wiki/ARRM_framework_introduction)

Original comments:

Comment 1: 

How are we defining "Accessibility" and "accessibility issue"/"issue"? They are interchanged and it jumps into responsibility of the issues without defining them. I think "accessibility issue" could mean "issue that impacts disabled users" or "WCAG success criteria failure". It might be nice to add the definition in the introduction to help people new to accessibility audits and terms we are using. I think the more broad definition of "issues that impact a disabled user" might be best in this case.

Comment 2:

[ED-low]: Pondering if 'owns' is the right word. It is succinct and snappy, but I can't help but thing that 'responsible for' is more accurate.

[ED-med]: If this is an introduction to the resource, it could benefit from a clearer introduction. For example, "This resource helps you define lines of responsibility and governance for managing accessibility as part of a digital project. While accessibility specialist knowledge might be centralised within a team, ownership for creating accessible projects is diffuse across a project stream." etc - high level introduction.

[ED-med]: Who owns accessibility?: The first two questions are perhaps not the only questions. They presuppose that someone has identified that accessibility is a thing and that someone has successfully (and correctly) identified issues to be fixed.

[ED-med]: There is an element of overly colloquial language: “We are saying no”, “in order to win the game”, “fail to plan … plan to fail”, “doesn't it make more sense… We believe so, too”.

[ED-med]: Using ARRM: I am not sure that this really describes how to use the resource – it is extremely vague on detail and possibly lends itself to an example.

Comment 3: 

Tone is much too lose “win the game” – accessibility is not a game. This overview is hard to scan and uses a lot of words before going into what ARRM is. This document needs significant work (especially the “Who owns accessibility” piece should probably condensed into a paragraph that says “ARRM is a framework to find out who is responsible for certain aspects inside an accessible workflow… Continued.”

Comment 4:

Not exactly edits, although I would be happy to take an edit pass with your permission. My main point here is that this does not read like other WAI resource intros. I think we need to focus on this resource and how to use it rather than trying to boil the entire issue to provide a detailed explanation of the need for distributed responsibility. A sentence or two along the lines of "Experience shows that accessibility is sustainable only when responsibility is distributed across roles. This tool is meant to help you do that..." or something. With that in mind:

1. Cut the entire first section, start with "The ARRM Framework is designed..."
2. Also cut the section on "Using ARRM" and what a team does *not* need to have.
3. Rather than Where to Start, have a section called "Components of ARRM" or something like that. 

After a couple of more brief intro sentences use the bulleted list to describe the Decision Tree and Role Definitions.

In the time since you first envisioned this and the current [unfinished sentence].

Comment 5:

I worry about starting this page with these two questions... "When an accessibility issue arises, who should correct it? How do you decide who should correct the issue?". I don't think the ARRM should only be about when issues "arise" or are found in an evaluation. I think many people will use this when talking to various teams about starting a project as well. If they can compare what they do at work to one of the roles, then they can look at that role's responsibility to see what they should be doing during planning, design, and development. The two questions are a great, clever way to catch people's attention in the intro. I would just be careful to not limit it to "issues that arise." This is the same for the first sentence under the "ARRM" heading. What you have under the "Using ARRM" section is more accurate to the use cases.

Resolution:

[Jenn] Suggested edits below.

**Comment by JenniferChadwick** [comment]
Suggested edits to current text:

[start of document]

"Experience shows that accessibility is sustainable only when responsibility is distributed across roles so everyone plays a role in the successful implementation of an inclusive culture and practice that takes into account the needs and preferences of all users, including those with disabilities.”

**What is the purpose of this resource?**

This resource helps you define lines of responsibility and governance for managing web accessibility as part of a digital project. While accessibility specialist knowledge might be centralized within a team, ownership for creating accessible projects is diffuse across a project stream.

**When an accessibility issue arises and needs a resolution, who should correct it?**

An “accessibility issue” is content or functionality within your website or software application that is impacting users with disabilities.  It may also signify a failure to comply with WCAG success criteria.

**How do you decide who should correct the issue?**

Accessibility issues must be addressed and resolved in order to remove barriers to the experience, for all users.  But who should correct the issue, and how you decide should be responsible for the solution - these questions are fundamental to creating accessible products of any kind.
 
When accessibility issues are technical in nature, they are typically treated as software bugs and assigned to a front-end developer. But should developers really be assigned tasks such as:

•	Selecting colors
•	Defining keyboard operation or global navigation approaches
•	Identifying headings
•	Authoring instructions or describing images…._

**What is the definition of “ownership”?**
 
[taken from the Role-based Decision Tree page] Using this framework, a team should more easily define who ultimately owns specific requirements for ensuring the accessibility of a website or software application and who else is actively involved in a supporting role – i.e., those that contribute to the successful implementation of the checkpoints in the lifecycle.

“Ownership” means having responsibility for completing the tasks related to the remediation or resolution of an accessibility issue, and the accountability to ensure that this responsibility is taken and the task is done, not just once, but regularly and consistently over time. 

**Using ARRM**

Web accessibility is all about planning. The most successful teams and organizations understand that in order to  implement design, development, writing and testing practices that regularly consider and include the needs and preferences of users with disabilities – these needs must be addressed as early as possible in the process.  This oftentimes starts as early as the design phase, if not earlier.
 
After all, it is now a well-established fact that teams and organizations which explicitly fail to plan for Web accessibility are typically the ones who implicitly plan to fail at creating accessible content.

**Comment by dboudreau** [comment]
Reflects changes that were made to the Introductory page, based on the feedback received.
Document is here: https://www.w3.org/WAI/EO/wiki/ARRM_framework_introduction

Anyone interested in providing additional feedback based on the latest version, please let us know in the comments below.

**Comment by mgifford** [comment]
@JenniferChadwick do you think we can close this now?
